services/one-stream/package/buildAnyPackage.js

```diff
@@ -256,12 +256,17 @@ export const buildAnyPackage = async (packageId, config) => {
         if (seaToolStatus && SEATOOL_TO_ONEMAC_STATUS[seaToolStatus]) {
           const oneMacStatus = SEATOOL_TO_ONEMAC_STATUS[seaToolStatus];
           putParams.Item.currentStatus = oneMacStatus;
-          putParams.Item.finalDispositionDate =
-            finalDispositionStatuses.includes(oneMacStatus)
-              ? DateTime.fromMillis(anEvent.STATE_PLAN.STATUS_DATE).toFormat(
-                  "yyyy-LL-dd"
-                )
-              : emptyField;
+          if (finalDispositionStatuses.includes(oneMacStatus)) {
+            putParams.Item.finalDispositionDate = DateTime.fromMillis(
+              anEvent.STATE_PLAN.STATUS_DATE
+            ).toFormat("yyyy-LL-dd");
+            // have to add 18000000 seconds because of time zone
+            putParams.Item.finalDispositionTimestamp =
+              anEvent.STATE_PLAN.STATUS_DATE + 18000009;
```

Comment:
I see what you are doing here since the stored date from seatool is an epoch timestamp with no time component and you are therefore forcing it to US Eastern UTC -5.
However you exposed the same flaw has existed on the original implementation converting it to yyyy-LL-dd format. Due to this I am seeing different dates on the table vs the details page for final disposition date since details still uses finalDispositionDate and not finalDispositionTimestamp